connector-j-5/src/main/java/com/google/cloud/sql/mysql/SocketFactory.java

```diff
@@ -83,15 +83,17 @@ public Socket afterHandshake() {
     return socket;
   }
 
-  // Returns True if running in a Google App Engine Standard runtime, otherwise False.
-  private boolean runningOnGaeStandard(){
+  // TODO(kurtisvg) move this check into a shared class
+  // Returns @{code true} if running in a Google App Engine Standard runtime.
```

Comment:
You need `/** ... */` for it to be Javadoc.

And for code snippets it's `{@code true}`.

Reply:
My mistake, fixed.